src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -65,7 +65,7 @@ An additional resource is the [Hardware Integration Toolkit](https://docs.google
 
 ### Using Status Data IDs
 
-There is an extensively defined Status Data ID list which can be found at MyGeotab >  **Engine & Maintenance** > **Engine & Device…** > **Diagnostics**. For each piece of information, conversion parameters are also required. Conversions must be in the form as per the table below:
+There is an extensively defined Status Data ID list which can be found at [MyGeotab Diagnostics](https://docs.google.com/spreadsheets/d/1sy7IOhWmFoo40_I-ruOJO8bVTMLXqHa11d0oJtaoIcE/edit#gid=1133172080). For each piece of information, conversion parameters are also required. Conversions must be in the form as per the table below:
 
 | Status Data ID | Description | Multiplier | Offset | Unit |
```

Comment:
Since we have the explanation of Conversion and Offset in the README sheet, would it make sense to remove some of this section?

Reply:
The CAN protocol page has a similar section that can use some simplification: https://geotab.github.io/sdk/hardware/addon-protocols/can/#2---request-status-data-ids

Reply:
Both the CAN and RS232 page seem to have an "integration process" which provides instructions for contacting solutions engineering and a demonstration of how to use the conversion and offset information.  Would you like me to remove the explanation from the RS232 page entirely and leave the one on the CAN page? Also what did you have in mind for the simplification? Maybe removing some of the examples?

Reply:
You can remove the content from both RS232 and CAN page. Removing the examples would be good, cuz they look redundant to me

Reply:
So to confirm, you want me to remove the examples section from both the RS232 and CAN page?

Reply:
Yes please

Reply:
Not sure which part specifically, but you can make the change. I'll review and let you know if the change should not be made

Reply:
Yep they have already been made with that idea in mind. Should be good to go.